compose/src/commonMain/kotlin/ResourceCache.kt

```diff
@@ -0,0 +1,9 @@
+package com.juul.krayon.compose
+
+import androidx.compose.runtime.Composable
+
+// TODO: Figure out why this doesn't compile when marked internal instead of public
+public expect class ResourceCache
+
+@Composable
+internal expect fun rememberResourceCache(): ResourceCache
```

Comment:
I don't follow why this is called "remember"? It doesn't look like any of the implementations use the `remember` lambda, so how is this upholding the Compose _remember_ functionality?

Reply:
It was supposed to be using a `remember` block, or else the paint cache does nothing 😅 . Thanks for catching that.